docs/resources/processes.md.erb

```diff
@@ -16,7 +16,7 @@ A `processes` resource block declares the name of the process to be tested, and
 
 where
 
-* `processes('process_name')` must specify the name of a process that is running on the system
+* `processes('process_name')` specifies the of a process to check. If this is a string, it will be converted to a Regexp. For more specificity, pass a Regexp directly.
```

Comment:
I think this should be `specifies the name of a process to check` - looks like "name" was dropped accidentally.